### ISSUE TITLE
Difficult to resize main window on Gnome3

### ISSUE THREAD

**Issue by mray271** [eteran/nedit-ng#291]
On GNome3 (not sure about other window managers), the area around the window borders and corners where the mouse allows resizing are incredibly narrow and make resizing very challenging.  Diagonal resize is especially tricky and requires essentially single pixel accuracy with mouse to be able to get the diagonal resize cursor to become active.

Is there a setting in nedit-ng to change the width of the mouse selection around borders? 
If this is a QT-specific thing, or maybe a theme specific thing is there a way to modify this ? 
Have others also encountered this issue?

**Comment by eteran** [comment]
The window borders are usually managed by the window manager. Is NEdit-ng the only program where you've experienced the issue?

**Comment by eteran** [comment]
Can you try playing with this setting?

```
gsettings set org.gnome.mutter draggable-border-width 15
```

https://stackoverflow.com/questions/47404431/how-to-make-the-borders-of-a-window-wider-in-gnome-3-26

**Comment by mray271** [comment]
Would there be a config similar to 
```draggable-border-width```
in QT5 somewhere?  Here's what the qt5ct dialog offers but doesn't seem to have an obvious setting like this
![Screenshot from 2021-01-18 22-43-41](https://user-images.githubusercontent.com/24377848/104997620-ba693180-59de-11eb-8a1b-493eec0c0b14.png)

**Comment by eteran** [comment]
Hmm, I'm not much of a gnome user, but I'll see what I can find.

It sounds like this is likely a general Qt apps on Gnome3 issue

**Comment by mray271** [comment]
I've created a cross-listed ticket here: https://bugreports.qt.io/browse/QTBUG-90569
Feel free to add yourself as follower.

**Comment by eteran** [comment]
@mray271 Awesome, this will be good to figure out!

Thanks

**Comment by tksoh** [comment]
>  It seems perhaps just by having QT_QPA_PLATFORMTHEME=qt5ct is all that is needed. 

@mray271  I tried this on ubuntu 20.04 after installing qt5ct, but it does not seem to change anything. Which version of Qt are you on? Not sure it matter, but I am on Qt 5.14.2

**Comment by tksoh** [comment]
OK. I discovered my problem can be solved by choosing a different theme (I use LXDE):

   https://forum.lxde.org/viewtopic.php?t=37646
   http://lxlinux.com/themes.html

Some themes (for example, ClearLooks) provide a larger window handle for resizing at the bottom of the window. We can resize the width by dragging the lower right corner of the handle.

**Comment by mray271** [comment]
@tksoh , I had meant that by exporting the environment variable QT_QPA_PLATFORMTHEME=qt5ct would be all that was needed for qt5ct to become the manager of your QT5 themes.  It seems if you don't set QT_QPA_PLATFORMTHEME=qt5ct, then you can't use the qt5ct dialog to manage your QT5 themes and you have to do it "manually".

**Comment by mray271** [comment]
@tksoh  So I did install LXDE desktop environment in Fedora 33 -- just for comparison.  It's not the default theme in Fedora so needed to install it.   And I found that the draggable window margins are essentially the same as when qt5ct is in operation on Gnome3.  I used the "motif" QT5 theme to give nedit-ng the old-school nedit look.  

Both LXDE and qt5ct in Gnome3 are ssable but I'd say uncomfortably narrow.   Without qt5ct controlling the theme in Gnome3 it's very very frustrating to resize because the draggable border width seems like 1 pixel accuracy is needed with the mouse.  

With qt5ct controlling the theme in Gnome3 or using LXDE,  the draggable border width feels like 2 pixels instead of 1 pixel. 

There's simply no comparison with the ease of resizing my Gnome3 terminal windows using the command that @eteran  suggested.

`gsettings set org.gnome.mutter draggable-border-width 15`

15 pixels wide is somewhat overkill (very very easy to resize the windows) but if you are programming, you don't want to spend energy with micro precision mouse movements.  You just want to resize your window and get on with things.
